doc/classes/InputEventAction.xml

```diff
@@ -35,8 +36,10 @@
 	</methods>
 	<members>
 		<member name="action" type="String" setter="set_action" getter="get_action">
+			Action's name. Each action is assigned and retrived by this [String].
```

Comment:
"The action's name." sounds a little better.

"Actions are accessed via this [String]."